Source/Controller/MenuViewController.swift

```diff
@@ -247,10 +247,6 @@ private class MenuView: UIView {
         let maxHeightConstraint = connectedPeersView.heightAnchor.constraint(equalToConstant: 310)
         maxHeightConstraint.priority = UILayoutPriority(501)
         maxHeightConstraint.isActive = true
-
-        if Date.todayIsAHoliday() {
```

Comment:
This was a cool Easter egg, but it has the old Planetary branding and only works with US holidays, so this Easter seemed like a good time to retire it.